src/fparser/two/utils.py

```diff
@@ -1038,62 +1037,84 @@ def tostr(self):
 
 
 class KeywordValueBase(Base):
-    """
-::
-    <keyword-value-base> = [ <lhs> = ] <rhs>
-    """
+    '''
+
+    keyword-value-base is [ <lhs> = ] <rhs>
```

Comment:
Could you please also remove the remaining <>. The idea is to work towards having any rules have the same format as the spec document.

Reply:
Done.